### PR TITLE
Release 0.3.1

### PR DESCRIPTION
**Pull request by PatelUtkarsh**
## Changelog:
Fix onboarding wizard. (See #160)

## Props 
Thanks to the many contributors who made this release possible through work on development, design, testing, project management, and more:

> @rodydavis, @pbearne, @PatelUtkarsh, @emeaguiar, @csossi, @jauyong, @mattchungxwp

### CODE REVIEW

**Comment by PatelUtkarsh** [comment]
@googlebot I consent.

File: plugin/assets/src/wizard/components/notice/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
 const Notice = props => {
 	return (
 		<div className={ `notice material-wizard__notice ${ props.type }` }>
-			<p>{ props.message }</p>
+			<p dangerouslySetInnerHTML={ { __html: props.message } }></p>
```
Comment (rodydavis) [inline plugin/assets/src/wizard/components/notice/index.js:25]: Why does this need to be dangerous set?

File: plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap [hunk 2]
```diff
@@ -8,9 +8,13 @@ exports[`Notice matches snapshot 1`] = `
   <div
     className="notice material-wizard__notice notice-error"
   >
-    <p>
-      There was an error
-    </p>
+    <p
+      dangerouslySetInnerHTML={
```
Comment (rodydavis) [inline plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap:12]: Why does this need to be dangerous set? Is there another way to approach this?

**Review by rodydavis — CHANGES_REQUESTED** [review]
@PatelUtkarsh please take a look

**Comment by rodydavis** [comment]
@pbearne i need a consent from you for the CLA

**Comment by pbearne** [comment]
@googlebot I consent.

File: plugin/assets/src/wizard/components/notice/index.js (hunk 1 above)
Comment (PatelUtkarsh) [reply plugin/assets/src/wizard/components/notice/index.js:25]: Hey @rodydavis, We are sending an error with the anchor tag to set the API key from the material setting page when the user clicks update font button:

We can: 
1. Remove the link and keep it a text error message (and also remove `dangerouslySetInnerHTML`).
1. Use the library to sanitize HTML such that it only allows specified tags. https://github.com/cure53/DOMPurify 

We are sanitizing it from PHP before passing it to the frontend here: https://github.com/material-components/material-design-for-wordpress/pull/165/files#diff-fd8d3142c003529c626daa8443ca0b86e5d6e29bf3d9a8cf4d3053d2cf65ba5aR225-R232

Comment (rodydavis) [reply plugin/assets/src/wizard/components/notice/index.js:25]: Yeah lets do option 2

File: plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap (hunk 2 above)
Comment (rodydavis) [reply plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap:12]: Lets sanitize

**Comment by rodydavis** [comment]
@PatelUtkarsh requested changes

**Comment by emeaguiar** [comment]
@googlebot I consent.

File: plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap (hunk 2 above)
Comment (rodydavis) [reply plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap:12]: @PatelUtkarsh was this missed? Should be sanitized or removed

**Review by rodydavis — CHANGES_REQUESTED** [review]
@PatelUtkarsh one final change

File: plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap (hunk 2 above)
Comment (PatelUtkarsh) [reply plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap:12]: Hey @rodydavis, This snap test is being generated after sanitization which we've added here: https://github.com/material-components/material-design-for-wordpress/pull/165/files#diff-28d761d68fde552ff61fcfc790098f46fe998345dc66b22fec33f4faebe0653eR32.

**Review by rodydavis — APPROVED** [review]
LGTM

**Pull request merged**